core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -137,6 +137,12 @@
      */
     TableStatistics getTableStatistics(Session session, TableHandle tableHandle, Constraint constraint);
```

Comment:
Maybe we can add a `previousStatisticsEstimate` argument to `getTableStatistics()` (from the old node being pushed-down) to allow the connector return the "best" estimate using the old and the new ones?
This way, there is no need to expose accuracy degradation comparison API.

Reply:
How would connector know whether old ones are better than new ones, without knowing how the old ones were calculated?

Reply:
Got it, thanks!